File: cs/CoreWcfDavcnaBlagajna/Models/PoslovniProstor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Threading.Tasks;

namespace CoreWcfDavcnaBlagajna.Models
{
    [DataContract(Namespace = "http://si.feri.soa/davcnablagajna/data/poslovniprostor/1.0")]
    public class PoslovniProstor
    {
        [DataMember(Name = "PpId", Order = 1)]

        public string PpId { get; set; }

        [DataMember(Name = "DavcnaSt", Order = 2)]
         public string DavcnaSt { get; set; }

        [DataMember(Name = "Naslov", Order = 3)]
         public string Naslov { get; set; }
    }
}

File: java/davcna-blagajna-soap-eksplicitno/src/test/java/si/feri/soa/davcnablagajna/entity/PoslovniProstor.java
package si.feri.soa.davcnablagajna.entity;

import jakarta.xml.bind.annotation.XmlRootElement;
import jakarta.xml.bind.annotation.XmlType;

@XmlRootElement
@XmlType(name = "PoslovniProstor", 
    namespace = "http://si.feri.soa/davcnablagajna/data/poslovniprostor/1.0")
public class PoslovniProstor {

    private String ppId;
    private String davcnaSt;
    private String naslov;
    public String getPpId() {
        return ppId;
    }
    public void setPpId(String ppId) {
        this.ppId = ppId;
    }
    public String getDavcnaSt() {
        return davcnaSt;
    }
    public void setDavcnaSt(String davcnaSt) {
        this.davcnaSt = davcnaSt;
    }
    public String getNaslov() {
        return naslov;
    }
    public void setNaslov(String naslov) {
        this.naslov = naslov;
    }

}
